mlocal/frags/Makefile.stub

```diff
@@ -20,7 +20,10 @@ collect:
 check: $(go_OBJ)
 	@echo " CHECK golangci-lint"
 	$(V)cd $(SOURCEDIR) && \
-		golangci-lint run --build-tags "$(GO_TAGS)" ./...
+		(PATH=$(SOURCEDIR)/bin:$$PATH; \
+		 (command -v golangci-lint >/dev/null || \
+		  curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.15.0) && \
```

Comment:
this should use -b $(BUILDDIR_ABSPATH)/bin

```suggestion
		  curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(BUILDDIR_ABSPATH)/bin v1.15.0) && \
```

Reply:
That's a different directory, under the builddir instead of in the source directory.  That is typically wiped out for every build, so it would have to be redownloaded every time.  Are you sure you want that change?

Reply:
Right. I probably have a different workflow than most people, I don't clean that often...

I don't like the idea of adding files to the repository like this, but since it's in .gitignore, I guess there's no chance of someone accidentally committing this.